modules/accessing-microshift-cluster-remotely.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+:_content-type: CONCEPT
+[id="accessing-microshift-cluster-remotely_{context}"]
+= Accessing the MicroShift cluster remotely
+
+Use the following procedure to access the MicroShift cluster from a remote workstation. 
+
+.Prerequisites 
+
+* You have installed the `oc` CLI. 
+
+.Procedure 
+
+[NOTE]
+====
+The following procedure assumes that there are two users. The first user, `user@microshift`, is the user on the{product-title} host machine. `user@microshift` is responsible for setting up `op-system` machine so that it can be accessed by a remote user on a separate workstation.  
```

Comment:
```suggestion
The following procedure assumes that there are two users. The first user, `user@microshift`, is the user on the {product-title} host machine. `user@microshift` is responsible for setting up `op-system` machine so that it can be accessed by a remote user on a separate workstation.  
```